### ISSUE TITLE
Questions using jquery-ui interfere with bootstrap tooltips

### ISSUE THREAD

**Issue by jonatanschroeder** [PrairieLearn/PrairieLearn#9465]
As discussed here: https://github.com/PrairieLearn/PrairieLearn/pull/9462#issuecomment-1962097760. Questions that include any reference to jquery-ui (either directly or with elements like pl-order-blocks, or custom elements) cause a conflict between jquery-ui tooltips and bootstrap tooltips. This can be an issue in pages that use tooltips, like the manual grading page (rubrics). Currently, when manually grading a question that includes order blocks and setting the rubrics, the tooltips don't show up.

**Comment by nwalters512** [comment]
This will be solved when we move to Bootstrap 5 😛 who know when that'll happen though.

**Comment by SethPoulsen** [comment]
well...we moved to Bootstrap 5. Is it fixed now?

**Comment by nwalters512** [comment]
I don't think this can be considered fully fixed until we _completely_ move to Bootstrap 5 and stop using the jQuery `$.tooltip(...)` interface for tooltips.

**Comment by reteps** [comment]
I did a grep for `$.tooltip` and didn't see any mentions. @nwalters512 ?

**Comment by jonatanschroeder** [comment]
I just tested manually grading a question with order blocks, and tooltips work as expected.
